RELEASE.md

```diff
@@ -0,0 +1,9 @@
+## Steps to do a release
```

Comment:
nice, thanks for this !

Reply:
I am easy to forget, so I often document everything, I mean EVERYTHING! lol

Reply:
+1 on adding a doc for releases! Thank you @min-xu-ai